### PR TITLE
call magic setter for read only props

### PR DESCRIPTION
**Pull request by jakemac53**
fixes https://github.com/dart-lang/polymer-dart/issues/608

### CODE REVIEW

**Comment by jakemac53** [comment]
fyi this only actually affects polymer dart elements, I have added a test in the regular polymer package. It is a bit weird to have the code here but I think its the best place.

File: lib/src/notify_path.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+for details. All rights reserved. Use of this source code is governed by a
+BSD-style license that can be found in the LICENSE file.
+-->
+<script>
+  (function() {
+
+    Polymer.PolymerInterop = Polymer.PolymerInterop || {};
+
+    Polymer.PolymerInterop.notifyPath = function (path, value) {
+      if (this.__isPolymerDart__ && path.indexOf('.') == -1) {
```
Comment (sigmundch) [inline lib/src/notify_path.html]: might be worth adding a short comment here explaining what this is

Comment (sigmundch) [inline lib/src/notify_path.html]: (and why it is ok to not do it for longer paths)

**Comment by sigmundch** [comment]
lgtm

**Pull request merged**